setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = aiohttp-cors
+version = 0.7.0
+author = Vladimir Rutsky and aio-libs team
+author_email = vladimir@rutsky.org
+license = Apache-2.0
+description = CORS support for aiohttp
+long_description = file: README.rst
+url = https://github.com/aio-libs/aiohttp-cors
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
```

Comment:
3.4 is gone, 3.7 is supported.

Reply:
I just converted what we already had with an automated script  into setup.cfg and axded some manual fixes.